pinot-clients/pinot-java-client/src/main/java/org/apache/pinot/client/BrokerResponse.java

```diff
@@ -38,6 +39,126 @@ private BrokerResponse(JsonNode brokerResponse) {
     _exceptions = brokerResponse.get("exceptions");
     _selectionResults = brokerResponse.get("selectionResults");
     _resultTable = brokerResponse.get("resultTable");
+    _responseStats = ResponseStats.fromJson(brokerResponse);
+  }
+
+  public static class ResponseStats {
```

Comment:
This class should be moved to a separate file.

Reply:
Done.